src/components/MapDisplay/MapDisplay.js

```diff
@@ -11,6 +23,7 @@ function MapDisplay({region, onSelectOutlet, selectedOutlet}) {
   
   const [downstream, setDownstream] = useState(null);
   const [mapDataset, setMapDataset] = useState(null);
+  const [datasetMinMax, setDatasetMinMax] = useState({});
```

Comment:
**Somewhat superseded: see comment above.**

`datasetMinMax` is a configuration item, and which one is in use is selected by which dataset is in use. So `datasetMinMax` isn't really state, it's just a derived value from whatever state value holds the current dataset selection (`mapDataset` I think maybe?). It would be better to just pull it out of config directly than to set it as a state variable and pass it in.